index.js

```diff
@@ -10,32 +11,35 @@ app.message('hello', async ({ message, say }) => {
   await say(`Hey there <@${message.user}>!`);
 });
 
-// WorkflowStep インスタンスを生成
-const ws = new WorkflowStep('new_data', {
+const workflowStep = new WorkflowStep('new_data', {
   // edit：ステップをワークフローに追加する際に実行
   edit: async ({ ack, step, configure }) => {
-    await ack();// Slack からイベントを受信したことを確認するのにack関数は必須で呼び出す
-    await configure({ blocks });// blocksを元にステップ設定モーダルをオープンする
+    await ack();
+    await configure({ blocks });
   },
 
-  // save：モーダルから値が送信されると実行される
+  // save：ワークフローステップモーダルから値が送信されると実行される
   save: async ({ ack, step, view, update }) => {
     await ack();
 
     const { values } = view.state;
     const name = values.task_name_input.name;
     const description = values.task_description_input.description;
     const captain = values.task_captain_input.captain;
-    const member = values.task_member_input.member;
+    const member1 = values.task_member1_input.member1;
+    const member2 = values.task_member2_input.member2;
```

Comment:
```suggestion
    const member1 = values.task_member1_input.member1;
    const member2 = values.task_member2_input.member2;
```

1, 2 だけだと順番を担保できないので明示したいです
e.g. 1st, 2nd

Reply:
修正中。